### DISCUSSION TITLE
How to use monaco-languageclient to implement notebook

### DISCUSSION THREAD

**Discussion by FoSuCloud** [TypeFox/monaco-languageclient#668] [Announcements]
I don't know if this configuration is correct, and how to configure the model for each cell
<img width="685" alt="截屏2024-05-21 18 50 10" src="https://github.com/TypeFox/monaco-languageclient/assets/49218295/8acad372-1c17-44ed-bdf1-c1d21f1e3c0c">

**Comment by kaisalmen** [comment]
Hi @FoSuCloud we don't have any notebook examples, yet. Did you try already try to alter an existing example and see if it works?

**Comment by zhanba** [comment]
I have implemented an language client in [libro](https://github.com/difizen/libro) notebook，and it reference some code from monaco-language-client.

**Comment by FoSuCloud** [comment]
Yes, I tried it, but unfortunately, it looks like there is no way to do it in the scheme
When implementing notebook with ="inmemory", must it be bound to the server file?

**Reply by kaisalmen** [reply]
@FoSuCloud is it possible to share what you already have? Or, are you willing to provide an new notebook example as a Draft PR and we try to fix this during review? This way you may get a solution and we can use it to as another example and use it to answer future questions from others.
